cmd/gobgpd/main.go

```diff
@@ -142,10 +145,27 @@ func main() {
 	maxSize := 256 << 20
 	grpcOpts := []grpc.ServerOption{grpc.MaxRecvMsgSize(maxSize), grpc.MaxSendMsgSize(maxSize)}
 	if opts.TLS {
-		creds, err := credentials.NewServerTLSFromFile(opts.TLSCertFile, opts.TLSKeyFile)
+		// server cert/key
+		cert, err := tls.LoadX509KeyPair(opts.TLSCertFile, opts.TLSKeyFile)
```

Comment:
why `tls.LoadX509KeyPair` is better than `credentials.NewServerTLSFromFile`?

Reply:
`credentials.NewServerTLSFromFile` does not give us access to the `*tls.Config` that is created as part the call for which we later possibly want to set the client CA certificate. This is at least the easiest way to code it in my knowledge.

`tls.LoadX509KeyPair` and then creating a `*tls.Config` is exactly the same as [`credentials.NewServerTLSFromFile` is coded to work](https://github.com/grpc/grpc-go/blob/f2fbb0e07ebf3dd46aa641ee89c9f17e8083eaf6/credentials/tls.go#L187-L192).